### ISSUE TITLE
conda.anaconda.org/anaconda channel not updated since 10/22/2020

### ISSUE THREAD

**Issue by trionfia** [ContinuumIO/anaconda-issues#12263]
Apologies if this is the wrong place for this.  I set up my conda package manager to use the package channels at the conda.anaconda.org web domain.  My understanding is conda.anaconda.org/anaconda is a mirror of repo.anaconda.com/pkgs/main.  However, the conda.anaconda.org/anaconda repodata.json file was last updated on 10/22/2020 while the repo.anaconda.com/pkgs/main repodata.json file was last update on 2/4/2021 (see screen captures provided).  Is the conda.anaconda.org/anaconda repo still meant to be an active mirror of repo.anaconda.com/pkgs/main?  If so, it looks like the replication job is not working properly.

![conda-anaconda-org](https://user-images.githubusercontent.com/46574174/106962572-5009fe00-670d-11eb-9552-0cb175fbae32.JPG)

![repo-anaconda-com](https://user-images.githubusercontent.com/46574174/106962711-85aee700-670d-11eb-875c-fa9768ccb8aa.JPG)

**Comment by trionfia** [comment]
@csoja is there any hope that this will get resolved?

**Comment by csoja** [comment]
yes - but unfortunately it is not a top priority at the moment.  Is there any reason you cannot use repo.anaconda.com? This is the preferred method of getting packages from Anaconda.  Beyond this bug, there are a few other issues with the anaconda.org/anaconda channel that also need to be fixed and are likely to impact a user's experience.

**Comment by trionfia** [comment]
Thanks for the reply @csoja .  I am doing work at a company that takes license compliance very seriously.  When Continuum changed the license model for the use of repo.anaconda.com, we were excluded from fetching packages from the repo.anaconda.com repository channels.  The license terms for the Anaconda Cloud repos like conda-forge and the anaconda mirror were deemed acceptable by our IT lawyer so we have been using only Anaconda Cloud repos.  I can switch to only conda-forge, but it is troublesome when dealing with upgrading Anaconda packages in base.
